mods/cnc/maps/nod06a/map.yaml

```diff
@@ -0,0 +1,819 @@
+MapFormat: 7
+
+RequiresMod: cnc
+
+Title: Steal The Detonator (a)
+
+Description: GDI has imported a Nuclear Detonator in an attempt to sway a few local political leaders. Penetrate the base and steal the detonator. A chopper will be sent to meet you at a designated landing zone. Look for the landing flare once you have stolen the device.
+
+Author: Westwood Studios
+
+Tileset: DESERT
+
+MapSize: 64,64
+
+Bounds: 16,17,46,45
+
+Visibility: MissionSelector
+
+Type: Campaign
+
+Videos:
```

Comment:
It already does (see [here](https://github.com/OpenRA/OpenRA/blob/bleed/OpenRA.Mods.Common/UtilityCommands/LegacyMapImporter.cs#L529-L555)). The map was probably imported before it did.